addons/mrp/models/mrp_workcenter.py

```diff
@@ -409,3 +415,19 @@ def _check_open_time_ids(self):
     def button_block(self):
         self.ensure_one()
         self.workcenter_id.order_ids.end_all()
+
+
+class MrpWorkCenterCapacity(models.Model):
+    _name = 'mrp.workcenter.capacity'
+    _description = 'Work Center Capacity'
+    _check_company_auto = True
+
+    workcenter_id = fields.Many2one('mrp.workcenter', string='Work Center', required=True)
+    product_id = fields.Many2one('product.product', string='Product', required=True)
+    product_uom_id = fields.Many2one('uom.uom', string='Product UoM', related='product_id.uom_id')
+    capacity = fields.Float('Capacity', default=1.0, help="Number of pieces of this product that can be produced in parallel.")
```

Comment:
English cleanup:

Number of pieces that can be produced in parallel for this product.